selfdrive/monitoring/dmonitoringd.py

```diff
@@ -22,12 +22,12 @@ def dmonitoringd_thread(sm=None, pm=None):
   sm['liveCalibration'].calStatus = Calibration.INVALID
   sm['liveCalibration'].rpyCalib = [0, 0, 0]
   sm['carState'].vEgo = 0.
-  sm['carState'].cruiseState.enabled = False
   sm['carState'].cruiseState.speed = 0.
   sm['carState'].buttonEvents = []
   sm['carState'].steeringPressed = False
   sm['carState'].gasPressed = False
   sm['carState'].standstill = True
+  sm['controlsState'].enabled = False
```

Comment:
this is already the default

Reply:
kk. can you fix the force decel in long test?